frontend/components/table/component.tsx

```diff
@@ -96,10 +96,7 @@ export const Table: FC<TableProps> = ({
 
                     // canSort is always true when manualSortBy is true
                     // See: https://github.com/TanStack/table/issues/2599
-                    const canSort =
-                      (sortingEnabled &&
-                        columns.find(({ accessor }) => accessor === id)?.canSort) ??
```

Comment:
The accessor may not have been set, and I'm not sure why I've implemented it this way since we have a `column` object already.